### ISSUE TITLE
Extract Image data to a separate table

### ISSUE THREAD

**Issue by Usurer** [Usurer/gallery#23]
Right now I store both Image and Folder data in a single table. This is easy to use, but results in a nullable Width, Height and Extension fields - which, obviously, don't exist for a folder.
I want to create a separate `Image` table (don't like this generic name but okay) that would store these three fields and image meta.

**Comment by Usurer** [comment]
Current `FileSystemItems` table has the following fields:
```
Id [Int PK],
ParentId,
IsFolder,
Path, 
Name,
CreationDate,
Extension,
Width,
Height,
UpdatedAtDate
```
I want to create another table `Images`:
```
Id [Int PK],
FileSystemItemId [Int FK],
Width,
Height,
Extension
```
That leaves us with the question regarding video files, but we don't support these anyway, so maybe just don't add them to DB at all (maybe that's how it works already)
Also I'm thinking about the need to have both Id and FileSystemItemId - could we use just a single Id that would be a FK to FileSystemItems, but right now I don't see too many drawbacks from having both.

**Comment by Usurer** [comment]
Closed with PR #27
